src/main/app/src/pages/ScheduleActivityPage.js

```diff
@@ -62,15 +63,15 @@ const ScheduleActivityPage = ({isLoggedIn, accessToken, userId, activity, links,
     }
 
     return (
-    <Grid container spacing={3} className={classes.root}>
-      <Grid item xs>
-        <Grid container justify="center" >
-          {randomArray.map((element) => <Button key={element.key} onClick={() => { alertUpdateActivity(element) }} size ="large" color="secondary" variant="outlined" className={classes.radio}> {element.title} </Button>)}
-          </Grid>
+      <Grid container spacing={3} className={classes.root}>
+        <Grid item xs>
+          <Grid container justify="center" >
+            {randomArray.map((element) => <Button key={element.key} onClick={() => { alertUpdateActivity(element) }} size ="large" color="secondary" variant="outlined" className="mx-1"> {element.title} </Button>)}
+            </Grid>
```

Comment:
I have changed the latest code to be a Card, since it is more informative! Make sure to look at them! :)

Reply:
Ok, that's not a problem. When I merge with the master branch I will make sure to keep the cards implementation and remove this one.